tests/wasm/0dockerfile

```diff
@@ -0,0 +1,10 @@
+
+FROM chromedp/headless-shell:latest
```

Comment:
I should admit I'm not a fan of using Docker everywhere (although it can be very useful in certain situations).

Why should the tests run in a Docker container? Can they also run directly on the host? In my experience, Docker often adds quite a bit of (startup) overhead and consumes a significant amount disk space.
(Not directly a request for change, but for clarification).

Reply:
Yes, retrospect I agree. The rationale was that it would be easy to ensure consistent behavior both in the CI and for local developers using docker.  However, since the CI now uses Chrome directly, and for developers it's probably just as easy, if not easier, to install Chrome as it is to install docker, I think all the docker stuff can just come out.

This would mean developers wanting to run the wasm test suite would just need to have Chrome installed in the usual place for their system, which I think is not any worse a requirement than requiring docker.

I'll do this refactor along with the rest of the points mentioned here. The result will definitely be simpler.